src/ui/components/Icon/styles.scss

```diff
@@ -24,90 +28,90 @@
 }
 
 .Icon-fox {
-  background: url("./fox.svg") center no-repeat;
+  @include icon($name: "fox");
+
   background-size: cover;
 }
 
 .Icon-fox-light {
-  background: url("./fox-light.svg") center no-repeat;
-  background-size: contain;
+  @include icon($name: "fox-light");
 }
 
 .Icon-mozilla {
-  background: url("./mozilla.svg") center no-repeat;
-  background-size: contain;
+  @include icon($name: "mozilla");
+
   height: 32px;
   width: 112px;
 }
 
 .Icon-user {
-  background: url("./user.svg") center no-repeat;
-  background-size: contain;
+  @include icon($name: "user");
 }
 
 .Icon-star {
-  background: url("./star.svg") center no-repeat;
-  background-size: contain;
+  @include icon($name: "star");
 }
 
 .Icon-star-empty {
-  background: url("./star-empty.svg") center no-repeat;
-  background-size: contain;
+  @include icon($name: "star-empty");
 }
 
 .Icon-star-full {
-  background: url("./star-full.svg") center no-repeat;
-  background-size: contain;
+  @include icon($name: "star-full");
 }
 
 .Icon-star-half {
-  background: url("./star-half.svg") center no-repeat;
-  background-size: contain;
+  @include icon($name: "star-half");
 }
 
 .Icon-user-dark {
-  background: url("./user-dark.svg") center no-repeat;
-  background-size: contain;
+  @include icon($name: "user-dark");
 }
 
 .Icon-user-fill {
-  background: url("./user-fill.svg") center no-repeat;
-  background-size: contain;
+  @include icon($name: "user-fill");
 }
 
 .Icon-browse {
-  background: url("./bullet-list.svg") center no-repeat;
+  @include icon($name: "bullet-list");
+
   background-size: cover;
   height: 20px;
   width: 20px;
 }
 
 .Icon-themes {
-  background: url("./themes.svg") center no-repeat;
+  @include icon($name: "themes");
+
   background-size: cover;
   height: 45px;
   min-width: 37.3px;
   width: 37.3px;
 }
 
 .Icon-extensions {
-  background: url("./extensions.svg") center no-repeat;
+  @include icon($name: "extensions");
+
   background-size: cover;
   height: 44.7px;
   min-width: 46.3px;
   width: 46.3px;
 }
 
 .Icon-artistic-unicorn {
-  background: url("./artistic-unicorn.svg") center no-repeat;
-  background-size: contain;
+  @include icon($name: "artistic-unicorn");
+
   height: 198.6px;
   width: 394px;
 }
 
 .Icon-multitasking-octopus {
-  background: url("./multitasking-octopus.svg") center no-repeat;
-  background-size: contain;
+  @include icon($name: "multitasking-octopus");
+
   height: 199.3px;
   width: 366px;
 }
+
+.Icon-featured {
+  @include icon($name: "featured");
```

Comment:
Yes, previously there was a default height/width configured with a variable. I kept this behavior.